src/tools/compiletest/src/json.rs

```diff
@@ -280,7 +278,7 @@ fn push_actual_errors(
                     line_num: Some(span.line_start + index),
                     kind: ErrorKind::Suggestion,
                     msg: line.to_string(),
-                    require_annotation: true,
+                    require_annotation: !line.is_empty(),
```

Comment:
Suggestion [COMMENT 1/2]: can you leave a comment here for the "why is this permitted" (i.e. PR description)